sdk/analytics/src/test/java/com/klaviyo/analytics/networking/requests/BaseApiRequestTest.kt

```diff
@@ -0,0 +1,67 @@
+package com.klaviyo.analytics.networking.requests
+
+import com.klaviyo.analytics.DevicePropertiesTest
+import com.klaviyo.analytics.model.Profile
+import com.klaviyo.fixtures.BaseTest
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+internal abstract class BaseApiRequestTest<T> : BaseTest() where T : KlaviyoApiRequest {
```

Comment:
Abstract base class for API request tests. The tests in this class get applied to all API Request tests that inherit from here.